.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+      # Setup QEMU (If Docker is required)
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
+
+      # Setup Docker Buildx (If Docker is required)
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v3  
+
+      # Build and push Docker image (If Docker is needed for your app)
+      - name: Build and push Docker image
+        id: docker_build
+        uses: docker/build-push-action@v3
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)